src/app/config/showOffline.ts

```diff
@@ -0,0 +1,3 @@
+export interface ShowOffline {
+    showOffline: boolean;
```

Comment:
this should probably be marked `readonly`

And, should this live in `configuration.model.ts` if it's still needed?